### ISSUE TITLE
Disable field and stop yup validation.

### ISSUE THREAD

**Issue by TruongTuyen7599** [jquense/yup#1391]
I have an issue with when disabling the field and I need to stop yup validation for some field.

**Comment by jquense** [comment]
yup don't have a way to "disable" validation, generally in that case the correct thing is to not validate that field. OR if a field can be disabled but also invalid you should consider using a `when()` to conditionally toggle validations based on other fields or outside `context`
